xtrack/beam_elements/elements_src/wire.h

```diff
@@ -0,0 +1,71 @@
+#ifndef XTRACK_WIRE_H
+#define XTRACK_WIRE_H
+
+/*gpufun*/
+void Wire_track_local_particle(WireData el, LocalParticle* part0){
+
+    // Data from wire
+    double const wire_L_phy = WireData_get_wire_L_phy(el);
+    double const wire_L_int = WireData_get_wire_L_int(el);
+    double const wire_current = WireData_get_wire_current(el);
+    
+    double const wire_xma = WireData_get_wire_xma(el);
+    double const wire_yma = WireData_get_wire_yma(el);
+
+
+
+    //start_per_particle_block (part0->part)
+
+        // constants : EPSILON_0, MU_0, PI, C_LIGHT,
+
+        #if !defined( C_LIGHT )
+            #define   C_LIGHT ( 299792458.0 )
+        #endif /* !defined( C_LIGHT ) */
+
+        #if !defined( EPSILON_0 )
+            #define   EPSILON_0 (8.854187817620e-12)
+        #endif /* !defined( EPSILON_0 ) */
+
+        #if !defined( PI )
+            #define PI (3.1415926535897932384626433832795028841971693993751)
+        #endif /* !defined( PI ) */
+    
+        #if !defined( MU_0 )
+            #define MU_0 (PI*4.0e-7)
+        #endif /* !defined( MU_0 ) */
```

Comment:
you can get the constants from ```constants.h``` as explained [here](https://github.com/xsuite/xtrack/pull/128/files#r819892179)

I don't think that we have mu0 there, feed free to add it